### PR TITLE
Migrate VM-Template to NFS-Datastore

### PR DESCRIPTION
**Pull request by jablonskim**

### CODE REVIEW

**Comment by jablonskim** [comment]
recheck no bug

**Comment by jablonskim** [comment]
recheck no bug

File: ansible/vmware-create-template.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@
         stage: snapshot
       when: vm_role == "testbed"
 
+    - name: Migrate VM
+      include_role:
+        name: vmware-template
+      vars:
+        stage: migrate
+        vcenter_datastore_name: "NFS-Datastore"
```
Comment (sodar) [inline ansible/vmware-create-template.yml]: We should extract this into a playbook parameter.

File: ansible/vmware-create-template.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@
         name: vmware-template
       vars:
         stage: migrate
-        vcenter_datastore_name: "NFS-Datastore"
+        vcenter_datastore_name: "{{ vcenter_datastore_name }}"
```
Comment (sodar) [inline ansible/vmware-create-template.yml]: You can remove this line, since `vcenter_datastore_name` is provided as extra variable.

**Pull request merged**